driver/normalizer/normalizer.go

```diff
@@ -243,3 +285,141 @@ var Normalizers = []Mapping{
 		},
 	)),
 }
+
+type singleQuote struct {
+	op Op
+}
+
+func (op singleQuote) Kinds() nodes.Kind {
+	return nodes.KindString
+}
+
+func (op singleQuote) Check(st *State, n nodes.Node) (bool, error) {
+	sn, ok := n.(nodes.String)
+	if !ok {
+		return false, nil
+	}
+	s := string(sn)
+	if !strings.HasPrefix(s, `'`) || !strings.HasSuffix(s, `'`) {
+		return false, nil
+	}
+	s = s[1 : len(s)-1]
+	s, err := unquoteSingle(s)
+	if err != nil {
+		return false, err
+	}
+	return op.op.Check(st, nodes.String(s))
+}
+
+func (op singleQuote) Construct(st *State, n nodes.Node) (nodes.Node, error) {
+	n, err := op.op.Construct(st, n)
+	if err != nil {
+		return nil, err
+	}
+	sn, ok := n.(nodes.String)
+	if !ok {
+		return nil, ErrUnexpectedType.New(nodes.String(""), n)
+	}
+	s := string(sn)
+	s = quoteSingle(s)
+	return nodes.String(s), nil
+}
+
+func unquoteSingle(s string) (string, error) {
+	// copied from strconv.Unquote
```

Comment:
Why copied?

Reply:
It needs to handle unescaping of `'` quotes.

Reply:
Can you please add a comment to that effect somewhere nearby?

Reply:
Done!